File: Python/Working with Numbers/28. Lowest Common Multiple.py
firstNumber = int(input("Enter the First Number:"))
secondNumber = int(input("Enter the Second Number:"))
number1 = firstNumber
number2 = secondNumber
hcf = 1

for i in range(1, min(firstNumber,secondNumber)+1):
    if((firstNumber % i == 0) and (secondNumber % i == 0)):
        hcf = i

lcm = (number1 * number2)/hcf

print(f"HCF of {firstNumber} and {secondNumber} is {lcm}")

File: Java/02. Working with Numbers/28. Lowest Common Multiple.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the First number:");
        int number1 = userInput.nextInt();
        
        System.out.println("Enter the Second number:");
        int number2 = userInput.nextInt();

        int originalNumber1 = number1;
        int originalNumber2 = number2;
        
        while( number1 != number2 ) {
            if( number1 > number2 ) {
                number1 -=  number2;
            } else {
                number2 -= number1;
            }
        }
        int lcm = (originalNumber1 * originalNumber2)/number1;
        System.out.println("The lCM is " + lcm);
        userInput.close();
    }
}